casanovo/denovo/evaluate.py

```diff
@@ -161,7 +161,12 @@ def aa_match(
     pep_match : bool
         Boolean flag to indicate whether the two peptide sequences fully match.
     """
-    if mode == "best":
+    if peptide1 is None and peptide2 is None:
+        return np.array([], dtype=bool), False
+    elif (peptide1 is None) != (peptide2 is None):
```

Comment:
**todo:** Slightly weird use of `!=`. Why not just use `or` instead, that's more legible. Then also the parentheses are not needed.

Reply:
I added this case before I added the `peptide1 is None and peptide2 is None`, so I was basically using the != as an XOR operator. I agree though that the or is definitely a lot more readable now that the first case is  `peptide1 is None and peptide2 is None`.